plugins/BEdita/Core/src/Model/Table/TreesTable.php

```diff
@@ -125,9 +128,39 @@ public function validationDefault(Validator $validator): Validator
         $validator
             ->boolean('canonical');
 
+        $validator
+            ->allowEmptyArray('params', null, static::jsonSchema(null) === true)
+            ->requirePresence('params', fn ($context) => $context['newRecord'] && static::jsonSchema(null) !== true)
```

Comment:
Same, but without a closure:

```suggestion
            ->requirePresence('params', static::jsonSchema(null) === true ? false : 'create')
```